cloud_governance/common/utils/utils.py

```diff
@@ -0,0 +1,5 @@
+
+
+class Utils:
+    pass
```

Comment:
Why we need it ?

Reply:
Another PR file, due to PR switching mistakenly added here.  Removed